Introcuced with version 1.0.0 and higher.

The config.h header file must be included for compilation fribidi, but not
during compilation Enigma2.
Error:

fribidi-types.h:31:11: fatal error: config.h: No such file or directory
|  # include <config.h>
|            ^~~~~~~~~~

The config.h header file is being generated by config.h.in and configure.ac, and is a temporary
file which doesn't end up in the recipe-sysroots directory.

Solution:
Undefine the HAVE_CONFIG_H macro to bypass the '#ifdef HAVE_CONFIG_H' checks which include the config.h 
header file.
This creates the exact behaviour as in version 0.19.7.

Possible fix in upstream (v1.0.6)?
https://github.com/fribidi/fribidi/commit/b534ab2642f694c3106d5bc8d0a8beae60bf60d3
See lib/fribidi-types.h file.
Applying of that backport patch breaks the build of fribidi for me (yet).
Also with all backport patches applied, so no go for now.

diff --git a/lib/gdi/font.cpp b/lib/gdi/font.cpp
index c06e7150e..b6706769b 100644
--- a/lib/gdi/font.cpp
+++ b/lib/gdi/font.cpp
@@ -26,6 +26,9 @@
 #include <lib/base/init.h>
 #include <lib/base/init_num.h>
 
+// undefine macro to fix build
+// against fribidi 0.19.7+
+#undef HAVE_CONFIG_H
 #include <fribidi/fribidi.h>
 
 #include <map>
